installutils/helmchart/manifests.go

```diff
@@ -73,7 +73,7 @@ func (m Manifests) CombinedString() string {
 	return buf.String()
 }
 
-var yamlSeparator = regexp.MustCompile("\n---")
+var yamlSeparator = regexp.MustCompile("\n---\n")
```

Comment:
i ran codegen in gloo and solo-projects and it didn't change the output.
let me see how difficult it would be to add tests here

Reply:
added tests under manifesttestutils 66a800d

i opted not to add additional tests under installutils because here the yaml split is done on [CombinedString](https://github.com/solo-io/k8s-utils/blob/5a61ba2ea8e90a9e4f1fc162a6b01d042f589659/installutils/helmchart/manifests.go#L79) which is [constructed with proper newlines](https://github.com/solo-io/k8s-utils/blob/5a61ba2ea8e90a9e4f1fc162a6b01d042f589659/installutils/helmchart/manifests.go#L69) so it's unlikely to produce the invalid yaml with missing newlines